### PR TITLE
Solc 0.5 0.6

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by andy8052 — APPROVED** [review]
Looks good to me

File: src/pause.sol [hunk 1]
```diff
@@ -51,7 +83,7 @@ contract DSPause is DSAuth, DSNote {
 
     // --- init ---
 
-    constructor(uint delay_, address owner_, DSAuthority authority_) public {
+    constructor(uint delay_, address owner_, address authority_) public {
```
Comment (lucas-manuel) [inline src/pause.sol:86]: Is it possible that this change will break other repos if they import this into their lib?

Comment (lucas-manuel) [reply src/pause.sol:86]: Everything else LGTM

Comment (gbalabasquer) [reply src/pause.sol:86]: Yes might break them, but that can be updated. I actually prefer to have any param type (and return values) as pure `address`, forcing them to specific interfaces makes sometimes the composability annoying.

**Review by nanexcool — APPROVED** [review]
awesome :)

**Pull request merged**